corehq/apps/change_feed/pillow.py

```diff
@@ -98,3 +106,13 @@ def _get_doc_type_from_change(change):
         return change['doc']['doc_type']
     except KeyError:
         return None
+
+
+def _create_deleted_couch_doc(doc_id, doc_type, deleted_on):
+    try:
+        DeletedCouchDoc.objects.create(doc_id=doc_id,
+                                       doc_type=doc_type,
+                                       deleted_on=deleted_on)
+    except IntegrityError:
+        # if it already exists, ignore it
```

Comment:
can you think of any occasion where a change would be picked up to `is_deletion` on an already deleted doc? would we want to know if this keeps happening?

Reply:
I was actually just talking to Cal about how pillows are designed to be idempotent, so the straightforward example for your question is this pillow being rewound to an earlier checkpoint and processing this change again. That was the main reason I chose to just ignore it entirely and not flag anything, since by design pillows should be able to handle this.

Reply:
got it 👍